ts/components/messages/MessagesInbox.tsx

```diff
@@ -106,15 +93,22 @@ const ListEmptyComponent = (paddingForAnimation: boolean) => (
     <Text style={styles.emptyListContentSubtitle}>
       {I18n.t("messages.inbox.emptyMessage.subtitle")}
     </Text>
-    {paddingForAnimation && <View style={styles.paddingForAnimation} />}
   </View>
 );
 
-/**
- * A component to render a list of visible (not yet archived) messages.
- * It acts like a wrapper for the MessageList component, filtering the messages
- * and adding the messages selection and archiving management.
- */
+// If have error in pot and the list is empty
+const ErrorLoadingComponent = () => (
```

Comment:
Since this component seems to be the same for all tabs, could it be defined in a dedicated file?